### DISCUSSION TITLE
auto ignore Salt connection Alerts

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2648] [Ideas]
I'm seeing a lot of 'sudo root' alerts that look like they're nothing more than the automated Salt checkins by from the Sensors to the Manager.  Shouldn't that particular Alert be ignored with an auto-created BFP rule during setup since it's a known connection setup by the Admin?
![Screenshot from 2021-01-15 11-06-24](https://user-images.githubusercontent.com/20424476/104764664-f2455000-5724-11eb-88cf-f071fec002ab.png)

**Comment by weslambert** [comment]
You can filter these out from the Wazuh alerts by modifying the Wazuh rules.  However, we may need to consider making a default filter/modified rule for these.

**Reply by innovate-support** [reply]
Ok, can someone supply an example of what to edit and the syntax of the rule?  I've read this page https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=suppression#suppressions but it's a bit over my head.  I just want to filter out all the 'normal' ssh salt traffic between the Sensor IP and the Manager IP.

**Reply by innovate-support** [reply]
back in the Squert days I could add 'ignore' rules right in the web interface and filter out a lot automatically.  It would be nice to be able to do that with SOv2
